apps/web/src/dialogs/command-palette/commands.ts

```diff
@@ -440,6 +444,26 @@ function getEditorCommands(): Command[] {
     });
   }
 
+  if (isFocusMode) {
+    commands.push({
+      id: "switch-to-normal-mode",
+      title: strings.normalMode(),
+      icon: FocusMode,
+      action: () => useAppStore.getState().toggleFocusMode(),
+      group: strings.editor(),
+      type: "command"
+    });
+  } else {
+    commands.push({
+      id: "switch-to-focus-mode",
+      title: strings.focusMode(),
+      icon: NormalMode,
+      action: () => useAppStore.getState().toggleFocusMode(),
+      group: strings.editor(),
+      type: "command"
+    });
+  }
+
```

Comment:
We can use a single command here: `Toggle focus mode`.

Reply:
I've opted to keep the conditional icon

```ts
 icon: isFocusMode ? FocusMode : NormalMode,
 ```
 
 Let me know if the icon should be static, or is this fine